src/sonar_widget/SonarPlot.cc

```diff
@@ -31,7 +31,7 @@ void SonarPlot::setData(const base::samples::Sonar& sonar)
     if (!sonar.beam_count || !sonar.bin_count)
         return;
 
-    sonar.beam_count > 1 ? isMultibeamSonar = true : isMultibeamSonar = false;
+    isMultibeamSonar = sonar.beam_count > 1 ? true : false;
```

Comment:
`sonar.beam_count > 1` is already a boolean expression.

`isMultiBeamSonar = (sonar.beam_count > 1)`